README.md

```diff
@@ -1,31 +1,37 @@
 # code-style
 
-Code style guidelines and rules for OpenZeppelin projects.
+Code style guidelines and rules for OpenZeppelin projects. Reusable linter and formatter configurations.
 
 ## Solidity
 
 To check the code style of smart contracts written in Solidity, we use
 [solium](https://github.com/duaraghav8/Solium) v1.1.7, with the rules in
 [.soliumrc.json](.soliumrc.json).
 
-## Javascript
+## JavaScript
 
-To check the code style of code written in Javascript, we use
-[eslint](https://eslint.org/), with the rules in
-[.eslintrc](.eslintrc) and the plugins:
+We use Prettier as a code formatter to keep our code consistent and to reduce friction for reading and reviewing code.
 
-- eslint-config-standard
-- eslint-plugin-standard
-- eslint-plugin-import
-- eslint-plugin-node
-- eslint-plugin-promise
-- eslint-plugin-react (only for react projects)
+We use Prettier through ESLint.
 
-## Prettier
+```
+npm install github:OpenZeppelin/code-style
+```
 
-We use `Prettier` as a code formatter to keep our code consistent and to reduce friction for reading and reviewing code.
+#### `.eslintrc`
+
+```
+{
+  "extends": ["./node_modules/code-style/.eslintrc"]
```

Comment:
Let's create [sharable conifg](https://eslint.org/docs/6.0.0/developer-guide/shareable-configs) for our Eslint. Something like `@openzeppelin/eslint-config-base`. Should be part of that PR but I would love to see it happen. What do you think?

Reply:
Yeah, that would be much nicer, but I was trying to avoid publishing to npm as this is mostly internal. I don't know if it makes that much sense to avoid that. What do you think?

We can also do it without publishing to npm but we would have to rename the repo/package to `eslint-config-openzeppelin` or do some other weird thing.

Reply:
What are the pros for publishing a shareable config rather than using the path to node modules as seen here?

Reply:
Two things:
* More semantically correct - we want to have dependencies on modules/packages rather than magic filepaths. If you forget to install `code-style` you would get an error that some filepath is missing. If you use `require` you would get an error that a package is not installed. 
* A bit related. Using `require` makes static analysis tools to be aware of `code-style` dependency. While if you use a filepath `code-style` may be marked as unused dependency. But that is theoretically speaking not practical at all at that moment.